perception/autoware_compare_map_segmentation/src/distance_based_compare_map_filter/node.cpp

```diff
@@ -76,6 +76,7 @@ bool DistanceBasedStaticMapLoader::is_close_to_map(
 bool DistanceBasedDynamicMapLoader::is_close_to_map(
   const pcl::PointXYZ & point, const double distance_threshold)
 {
+  std::lock_guard<std::mutex> lock(dynamic_map_loader_mutex_);
```

Comment:
Are you taking a mutex lock for the whole `is_close_to_map` function?

Reply:
@veqcc Thanks, I reduce scope of locking here https://github.com/autowarefoundation/autoware.universe/pull/10222/commits/7d8359a6b08b90ad1fdfd455ed8562232517245b